src/wallet/rpc/addresses.cpp

```diff
@@ -539,6 +539,10 @@ RPCHelpMan getaddressinfo()
                         {
                             {RPCResult::Type::STR, "label name", "Label name (defaults to \"\")."},
                         }},
+                        {RPCResult::Type::ARR, "use_txids", "",
+                        {
+                            {RPCResult::Type::STR_HEX, "txid", "The ids of transactions involving this wallet which received with the address"},
```

Comment:
Perhaps better to call the field `receiving_txids` or similar, to avoid the ambiguity between "I've received funds via this address" and "I've spent funds that I received via this address".

Reply:
I agree with @ajtowns, I confess it took me a while to understand what `use_txids` means, at a first look I thought it would return all txids that involves that address but reading the doc could understand that's only about the receiving ones. `receiving_txids` sounds better.